submissions/Eugene-CG/memory-pair-game/index.js

```diff
@@ -0,0 +1,71 @@
+const cardsList = document.querySelector(".cards__list");
+let clickRemember = [];
+class Card {
+  constructor(img) {
+    this.img = img;
+  }
+  createCard() {
+    return `
+    <li class="card__container">
+          <div class="card card-back">
+              <img src="./img/${this.img}">
+          </div>
+      </li>`;
+  }
+}
```

Comment:
No need for this `class` here. There is no inheritance, no logic inside, etc. Basically, you're using it for data, so just use objects :)